activities/resolution_hierarchy.py

```diff
@@ -184,40 +246,262 @@ def copy(var):
     args['res_lt_max'] = args['resolution'] < (args['resolution_max'] - 1)
     return args
 
-def make_args(args, start, stop, step, dim, use_iso_flag, index_annotations):
+def chunk(xs, size):
```

Comment:
Could use a comment explaining what this does.

Reply:
added comments